utils.py

```diff
@@ -1,6 +1,11 @@
-"""Standard utilies"""
+"""
+Standard utilies
```

Comment:
At the top of the file is where you describe what kinds of things belong in this file, if it is not immediately obvious. I think `Standard utilities` is sufficient to describe the kinds of things that will be in this file.

Documentation for specific functions does not belong at the top of the file; instead, it belongs inside each function, at the top of each function.